release_notes/ose_3_2_release_notes.adoc

```diff
@@ -0,0 +1,736 @@
+Authentication] for details.
+- The `SETUID` and `SETGID` capabilities have been added back to the *anyuid* SCC,
+which ensures that programs that start as root and then drop to a lower
+permission level will work by default.
+- Quota support has been added for `*emptydir*`. When the quota is enabled on an
+XFS system, nodes will limit the amount of space any given project can use on a
+node to a fixed upper bound. The quota is tied to the `*FSGroup*` of the
+project. Administrators can control this value by editing the project directly
+or allowing users to set `*FSGroup*` via SCCs.
+- The `*DaemonSet*` object is now limited to cluster administrators because pods
+running under a `*DaemonSet*` are considered to have higher priority than
+regular pods, and for regular users on the cluster this could be a security
+issue.
+- Administrators can prevent clients from accessing the API by their `User-Agent`
+header the new `*userAgentMatching*` configuration setting.
+
+==== Integrated Docker Registry
+
+- The integrated Docker registry now supports Azure Blob Storage, OpenStack Swift,
+and Amazon CloudFront as storage back ends.
+- A readiness probe and health check have been added to the integrated registry to
+ensure new instances do not serve traffic until they are fully initialized.
+
+==== Routes
+
+- You can limit the frequency of router reloads using the `--interval=DURATION`
+flag or `*RELOAD_INTERVAL*` environment variable to the router. This can
+minimize the memory and CPU used by the router while reloading, at the cost of
+delaying when the route is exposed via the router.
+- Routers now report back status to the master about whether routes are accepted,
+rejected, or conflict with other users. The CLI will now display that error
+information, allowing users to know that the route is not being served.
+
+==== Storage
+
+- The `*NoDiskConflicts*` scheduling predicate can be added to the scheduler
+configuration to ensure that pods using the same Ceph RBD device are not placed
+on the same node. See link:../admin_guide/scheduler.html[Scheduler] for details.
+
+==== Administrator CLI
+
+- The administrative commands are now exposed via `oc adm` so you have access to
+them in a client context. The `oadm` commands will still work, but will be a
+symlink to the `openshift` binary.
+- The help output of the `oadm policy` command has been improved.
+- Service accounts are now supported for the router and registry:
+** The router can now be created without specifying `--credentials` and it will use
+the router service account in the current project.
+** The registry will also use a service account if `--credentials` is not
+provided. Otherwise, it will set the values from the `--credentials` file as
+environment on the generated deployment configuration.
+- Administrators can pass the `--all-namespaces` flag to `oc status` to see status
+information across all namespaces and projects.
++
+WARNING: This will create a lot of output.
```

Comment:
Agree. It came over from the Origin notes, was trying to make it work, but it's weird. Removing.